lib/bgsite_official/categories/tag.ex

```diff
@@ -17,4 +17,20 @@ defmodule BgsiteOfficial.Categories.Tag do
     |> cast(attrs, [:title, :banner, :description, :likes])
     |> validate_required([:title, :banner, :description, :likes])
   end
+
+  def changeset_update_projects(%Tag{} = tags, attrs) do
+      website
+      |> cast(%{}, @required_fields)
+      # associate projects to the user
+      |> put_assoc(:websites, websites)
+  end
+
+  many_to_many
+  (
+      :websites,
+      Website,
+      join_through: "website_project",
```

Comment:
The join table (when created) will be `website_tag`, not `.._project`. We will generate a migration for the join table.